jest.d.ts

```diff
@@ -0,0 +1,7 @@
+declare namespace jest {
+  interface Matchers<R> {
+    toHaveProp(selector: string): void;
```

Comment:
Can we specify where those custom matchers come from?

```suggestion
    // The following matchers are part of https://www.npmjs.com/package/@commercetools/jest-enzyme-matchers
    toHaveProp(selector: string): R;
```

Also, the return type should be `R` and not `void` (e.g. https://github.com/facebook/jest/blob/ccea646e76efd73654468ac32b3cb3a88778f7f4/packages/expect/src/types.ts#L87-L343)